bbn-1: FP registry guide (#1)
babylonlabs-io/networks: bbn-1/finality-providers/README.md

staking web application. This web application provides a curated list of
finality providers for the stakers to choose from for their delegations.
Besides critical information such as the EOTS public key and the commission
rate, for each finality provider, the web application also displays
human-friendly information such as the moniker and the website URL.

This repository is the place for finality providers to register such
information and all the finality providers listed in this repository will be
visible in the web application. This guide instructs finality providers on how
to get listed.

## Registration Eligibility Criteria

For security and quality reasons, strict eligibility criteria are imposed,
focusing on the finality provider's proven commitment and identity
verification. More specifically, for a finality provider to be eligible for
inclusion in this information registry, it needs to meet the following
requirements:

- having participated in the
[Babylon testnet-4](https://github.com/babylonchain/networks/tree/ac531139d5a75e575b34a80c9f8fc841cc33adab/bbn-test-4),
- to go through a know your business (KYB) process conducted by Babylon Labs, and

> - to submit a pull request (PR), and the deadline is **August 16th, end of the

Reviewer: ```suggestion
- to submit a pull request (PR) before the deadline of **August 16th, end of the
```

finality provider can be identified with (e.g. your moniker). It should not
contain white spaces or unrecognizable characters.

Inside this file, store the following JSON information corresponding to the
finality provider.

```json
{
  "description": {
    "moniker": "<moniker>",
    "identity": "<identity>",
    "website": "<website>",
    "security_contact": "<security_contact>",
    "details": "<details>"
  },
  "eots_pk": "<finality_provider_eots_pk>",
  "commission": "<commission_decimal>"
}
```

Properties descriptions:

- `moniker`: nickname of the finality provider.
- `identity`: optional Keybase.io identity signature. It is used to retrieve
the finality provider icon, the same way Cosmos-SDK uses for validators.
- `website`: optional website link URL.
- `security_contact`: required email for security contact.
- `details`: any other optional detail information.
- `eots_pk`: the btc pub key in hex format (The `pub_key_hex` property of the
`eotsd keys add` command output).
- `commission`: the commission charged for BTC staking rewards.
The commission will be parsed as a decimal:
  - `"1.00"` represents 100% commission.
  - `"0.10"` represents  10% commission.

>   - `"0.01"` represents  01% commission.

Reviewer: Let's make this example the minimum commission rate.

Author: https://github.com/babylonlabs-io/networks/pull/1/commits/502fc3f824d3a163448dcdcea3fe89e29989b144

**⚠ Warning!**

> The minimum commission value accepted is 5%. It will remain immutable and can't

Reviewer: Let's wait until the decision for this is finalized.

Author: 3%, I will update

Author: https://github.com/babylonlabs-io/networks/pull/1/commits/502fc3f824d3a163448dcdcea3fe89e29989b144
